test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2642,6 +2642,36 @@ def forward(self, *tensor_list):
         x = torch.randn(3, 4)
         self.run_test(EinsumModelTranspose(), input=(x,))
 
+    @unittest.skip("Enable this once ORT version is updated")
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_mse_loss(self):
+        class MSELossNone(torch.nn.Module):
+            def forward(self, input, target):
+                loss = torch.nn.MSELoss(reduction='none')
+                return loss(input, target)
+
+        x = torch.randn(3, 5)
+        y = torch.randn(3, 5)
+        self.run_test(MSELossNone(), input=(x, y))
+
+        class MSELossSum(torch.nn.Module):
+            def forward(self, input, target):
+                loss = torch.nn.MSELoss(reduction='sum')
+                return loss(input, target)
+
+        x = torch.randn(3, 5, requires_grad=True)
+        y = torch.randn(3, 5, requires_grad=True)
+        self.run_test(MSELossSum(), input=(x, y))
+
+        class MSELossMean(torch.nn.Module):
+            def forward(self, input, target):
+                loss = torch.nn.MSELoss()
+                return loss(input, target)
+
+        x = torch.randn(3, 5, requires_grad=True)
+        y = torch.randn(3, 5, requires_grad=True)
+        self.run_test(MSELossMean(), input=(x, y))
```

Comment:
add a test case with requires_grad=False for the target? (and add a small comment explaining that the output graph is different if  requires_grad=True/False for the target)

Reply:
I meant add a test with requires_grad=False in test_pytorch_onnx_onnxruntime. The current test all have target.requires_grad=True so will not test the mse op

Reply:
there is already test when requires_grad=False